pennylane/fourier/reconstruct.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Contains a function that computes the fourier series equivalent to
+a quantum expectation value."""
+from functools import wraps
+from inspect import signature
+import warnings
+
+# import numpy as np
+import pennylane as qml
+from pennylane import numpy as np
```

Comment:
how come autograd is needed here?

Reply:
This is resolved due to moving to `qml.math` :)